src/app/address/googleAutocomplete/GoogleAutocompleteFormField.tsx

```diff
@@ -76,7 +79,7 @@ const GoogleAutocompleteFormField: FunctionComponent<GoogleAutocompleteFormField
         <div className={ `dynamic-form-field dynamic-form-field--addressLineAutocomplete` }>
             <FormField
                 input={ renderInput }
-                labelContent={ labelContent }
+                label={ <label className="form-label optimizedCheckout-form-label" htmlFor={ inputProps.id } id={ labelID }>{ labelContent }</label> }
```

Comment:
Can we use the `Label` component instead?

Reply:
Yes, I should use `Label` as much as possible. Will update you once I finish the test.